packet.py

```diff
@@ -0,0 +1,69 @@
+"""
+Input format: {1,2,3,4|sum} where 1 is the sensor ID, 2, 3, 4 is the data,
+and sum is the result returned from checksum.
+Packet object format: integer id, integer list data, and string checksum.
+"""
+
+
+class Packet:
+    def __init__(self, data):
+        self.sensor_id, self.data, self.checksum = self.decode_message(data)
+
+    def __init__(self, id, data):
+        self.sensor_id = id
+        self.data = data
+        encoding = str(id)
+        for i in data:
+            encoding += "," + str(i)
+        checksum = fletcher16(encoding)
+        self.checksum = checksum
```

Comment:
Move these into `encode_data` and here, just call encode data with the right parameters.
Add a string variable that holds the encoded message so after I create a packet instance with 
```
packet = Packet(3, {23, 43, 5})
packet.encoded_message
```